drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -209,6 +209,7 @@
 
 .xblock--drag-and-drop .zone {
     position: absolute;
+    border: 1px hidden #565656;
```

Comment:
@agaylard Now that you have the `zone-with-borders` class, I think you can remove this line.